src/api_midlevel.jl

```diff
@@ -46,6 +46,19 @@ function get_extent_dims(obj::Union{Dataspace,Dataset,Attribute})
     return dims, maxdims
 end
 
+"""
+    HDF5.get_extent_ndims(obj::Union{HDF5.Dataspace, HDF5.Dataset, HDF5.Attribute}) -> ndims
+
+Get the number of array dimensions from a dataspace, dataset, or attribute

```

Comment:
```suggestion
Get the number of array dimensions from a dataspace, dataset, or attribute.

```
Should we mention this is equivalent to `length(HDF5.get_extent_dims(obj)[1])`?

Reply:
> Should we mention this is equivalent to length(HDF5.get_extent_dims(obj)[1])?

I think that should be obvious, so I would say no.